ZOB Sabine MkII 2nd Upload (Now with Better Decals and Floor Tiles)

Rubeebeebee opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
2nd upload of the sabine MkII (due to technical difficulties on my end updating), now updated with better decals and floor tiles, for swagginess, removed 4 thrusters from the back, as per request

The Sabine MkII is a combat capable modification of the CIV class hull Sabine, modified and retrofitted by the Zealots of Balaam for it's use in both planetary expeditions and enacting the will of their prophet.  It's ill-advised modifcations include retrofitting the previous docking, cargo bay, and living space to be a fully fledged gun ports for it's new armanents: which include

1x 90mm Autocannon
2x Dravon 57mm mortars
2x 20mm PD machineguns 

due to these ill-advised and "safety overides" engineering of the Zealots of Balaam, half of the ship has to be exposed to the vacuum of space, much to the chagrin of the end users. 

While upgunned, the hull is still the same from the regular CIV class sabine, making it a fragile target in prolonged combat. 

It's strength, however, lies in it's agility, with an upgraded amount of forward thrusters, as well as the addition of two gyroscopes, it should be able to outrun and outmaneuver any larger ship classes it can come up against. 

It's guns are located in it's cargohold, which makes the guns point directly at the rear of the ship- whilst most people see this as a disadvantage, Baalamites use this to their gain, deploying the Sabine MkII in hit-and-run tactics against much stronger and slower foes.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->

I've made this to add a lightly armored, yet agile ship that's only really good for doing hit and run tactics, and you need atleast two crew, one to pilot, and one to man the gunnery console, because the guns are all pointed backwards, while the ship is pointed forwards. (theoretically anyone can just git gud by driving and shooting at the same time, but then they'd frag in any ship anyways)

it is heavily armed for it's size, but it's still using the same hull from the original sabine, meaning any shot could be debilitating. 

It is super agile though, with it's additional thrusters and security class gyroscopes. 


## How to test
<!-- Describe the way it can be tested -->
load the .yml in an IDE, it should work. 

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="828" height="806" alt="newsabinemk2 live" src="https://github.com/user-attachments/assets/8fd79067-0187-45cd-8007-b9a2d0e9ea52" />
<img width="814" height="795" alt="newsabinemk2" src="https://github.com/user-attachments/assets/8f104e76-0be0-48ff-a5a2-616030673b89" />




## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
- [X] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: The Sabine MkII, a zealot of balaam MIL class ship, made for hit-and-run tactics. 
- tweak: updated the decals and floor tile choices to be more pretty, also removed 2 thrusters from the back as requested

Rubeebeebee committed:
re-uploaded the sabine with better decals!

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into sabinemk2-2nd-upload

OneZerooo0 reviewed (changes requested):
<img width="454" height="497" alt="image" src="https://github.com/user-attachments/assets/223e68f3-f387-419f-a3e9-87bd7ec41bbb" />

Please tweak these windows to include less diagonals, they make the visibility weird

<img width="406" height="230" alt="image" src="https://github.com/user-attachments/assets/d8650d42-1f9a-4829-bc3e-b70d6895fed7" />


Tone down roundstart armory a ton, a kammerer and a melee weapon at most


Finally, it would be helpful for initial purchase (and also funny) to have the spaced deck area not-spaced on purchase

Rubeebeebee committed:
updated the bridge windows to be not diagonal, and updated the armory to just contain 1x kammerer and 1x breaching hammer

Rubeebeebee committed:
updated the bridge windows to be not diagonal, and the armory to only contain a kammerer, a breaching hammer, and two boxes of buckshot.

Rubeebeebee committed:
Merge branch 'Monolith-Station:main' into sabinemk2-2nd-upload

Rubeebeebee committed:
updated the bridge windows to be not diagonal, and the armory to only contain a kammerer, a breaching hammer, and two boxes of buckshot.  + a gnome

Rubeebeebee committed:
updated the bridge windows to be not diagonal, and the armory to only contain a kammerer, a breaching hammer, and two boxes of buckshot. also added a gnome plush

(theres some sorta weird fuck up with my files, but ive clicked unstage on all of them, hopefully they dont mess up this branch)

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
> <img alt="image" width="454" height="497" src="https://private-user-images.githubusercontent.com/172208291/470337326-223e68f3-f387-419f-a3e9-87bd7ec41bbb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMzODkxNjIsIm5iZiI6MTc1MzM4ODg2MiwicGF0aCI6Ii8xNzIyMDgyOTEvNDcwMzM3MzI2LTIyM2U2OGYzLWYzODctNDE5Zi1hM2U5LTg3YmQ3ZWM0MWJiYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyNFQyMDI3NDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYTExMmMxOGY2YjFkMzJiMzc5ZGVmMDE5NmQzYzM0OTY2YzBkMDk1OWM0N2I0NzY0ODQxZDQ1ZmM5MTdkZGI5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.70-v6VDZtZDKnItThA_7gi_xCfogYEX3awbpMPnZRbM">
> 
> Please tweak these windows to include less diagonals, they make the visibility weird
> <img alt="image" width="406" height="230" src="https://private-user-images.githubusercontent.com/172208291/470337624-d8650d42-1f9a-4829-bc3e-b70d6895fed7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMzODkxNjIsIm5iZiI6MTc1MzM4ODg2MiwicGF0aCI6Ii8xNzIyMDgyOTEvNDcwMzM3NjI0LWQ4NjUwZDQyLTFmOWEtNDgyOS1iYzNlLWI3MGQ2ODk1ZmVkNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyNFQyMDI3NDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZWRjMzI1ZGUxYmM3NTk2MjZkNjk5NjY4ZWNkNzkxYjA2MzMwNzY5YjNjY2U2ZGUwZjU1NTViNmJjMzY2YTEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.sRqIm77OVhB66qvRynC8Gg31Z52gvjlOOxSy_xniVaM">
> 
> Tone down roundstart armory a ton, a kammerer and a melee weapon at most
> 
> Finally, it would be helpful for initial purchase (and also funny) to have the spaced deck area not-spaced on purchase


<img width="698" height="535" alt="youvebeengnomed" src="https://github.com/user-attachments/assets/0d38d1c2-a2cd-4e40-a05a-8bca6f8d5b29" />
<img width="572" height="731" alt="sabine mk2 tweaks" src="https://github.com/user-attachments/assets/dd60c9d0-ea93-41a6-8252-3fb9c14b5eeb" />

updated the bridge windows to be not diagonal, and the armory to only contain a kammerer, a breaching hammer, and two boxes of buckshot. plus a gnome plushie 

and i would like to argue to keep the spaced compartment spaced, since i know someones gonna walk in that compartment without an eva suit and randomly open up a shutter, exposing them to the void. Keeping the area spaced communicates that "yes, that area is spaced, and will continue to be space regardless if the shutters are open or closed." so they dont accidentally walk in there without an eva suit.

Rubeebeebee commented:
OH SHIT WHY DOES IT SAY I CHANGED 700+ FILES WTF

Rubeebeebee reviewed (commented):
HOW DO I UNFUCK THIS AAAAAAAAA

Honestly101 commented:
seems you accidently merged incorrectly from main causing all other PR commits to be reverted to before you last updated your repo.

Rubeebeebee commented:
> seems you accidently merged incorrectly from main causing all other PR commits to be reverted to before your last updated.

seems like it! im gonna try to do git revert [69e9a34] to try and uh... fix this

Rubeebeebee committed:
i tried git revert HEAD hopefully this fixes it?

monolith8319 committed:
chore: automatically update REUSE headers

Rubeebeebee commented:
> seems you accidently merged incorrectly from main causing all other PR commits to be reverted to before you last updated your repo.

would you know how to revert a specific commit? The [69e9a34] seems to be the problem here

Rubeebeebee commented:
i swear this fucking ship is cursed, this is the second time this has happened

Rubeebeebee committed:
i did git revert -m 1 69e9a34050e687b6e92f9053b34e07b4eba9bfd1 on the terminal on visual studio, i pray this works

OneZerooo0 commented:
git log
Find the latest commit that didn’t fuck up’s hash
git checkout (hash of unfucked commit)
git reset —hard
Doing git reset —hard clears *everything* done after the commit you checked out so use it with care

Rubeebeebee commented:
> git log Find the latest commit that didn’t fuck up’s hash git checkout (hash of unfucked commit) git reset —hard Doing git reset —hard clears _everything_ done after the commit you checked out so use it with care

alright, ive inputted these commands in the uh, terminal
what now?

Rubeebeebee commented:
> git log Find the latest commit that didn’t fuck up’s hash git checkout (hash of unfucked commit) git reset —hard Doing git reset —hard clears _everything_ done after the commit you checked out so use it with care

right, couldnt figure out how to fix THIS branch, so i just made a new branch/PR

https://github.com/Monolith-Station/Monolith/pull/1608

OneZerooo0 commented:
Newer PR exists, plus conflict and issues
